app/containers/AudioPlayer/PlayButton.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { TouchableOpacity } from 'react-native';
+
+import { CustomIcon } from '../CustomIcon';
+import { useTheme } from '../../theme';
+import styles from './styles';
+import RCActivityIndicator from '../ActivityIndicator';
+import { AUDIO_BUTTON_HIT_SLOP } from './constants';
+
+interface IButton {
+	disabled?: boolean;
+	onPress: () => void;
+	audioState: TAudioState;
+}
+
+type TCustomIconName = 'arrow-down' | 'play-shape-filled' | 'pause-shape-filled';
+
+export type TAudioState = 'loading' | 'paused' | 'to-download' | 'playing';
+
+const IconToRender = ({ audioState, disabled }: { audioState: TAudioState; disabled: boolean }) => {
```

Comment:
Redundant. If it's a component, the goal is to render.
You can just call it `Icon` on this scope without interfering other places.